python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -773,6 +790,11 @@ export default function EditorContainer({
       )}
       <Container maw="80rem">
         <Flex justify="flex-end" mt="md" mb="xs">
+          <div>
+            <Button loading={undefined} onClick={onClearOutputs}>
+              Clear Outputs
+            </Button>
+          </div>
```

Comment:
I noticed save button had updated styling. Not sure what is required for this button.

Reply:
Not related to this diff, but let's update "save" button colour to be green or blue, definitely not red